pegasus/sites.v3/code.org/views/dance_teacher_resources.haml

```diff
@@ -1,20 +1,31 @@
 %link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/dance-landing/dance-teacher-resources.css'}
 
 :ruby
+  locale = I18n.locale.to_s.downcase
+    translated_languages = ["es-mx", "es-es", "fr-fr", "ko-ko", "tr-tr", "zh-tw"]
+    lesson_link = translated_languages.include?(locale) ?
+    "https://curriculum.code.org/#{locale}/hoc/plugged/8" :
+    "https://curriculum.code.org/hoc/plugged/8"
+    unplugged_link = translated_languages.include?(locale) ?
+    "https://curriculum.code.org/#{locale}/hoc/unplugged/4" :
+    "https://curriculum.code.org/hoc/unplugged/4"
+
```

Comment:
tiny style nits: `translated_languages`, `lessson_link` and `unplugged_link` should align with `locales`.